src/components/NavBar/index.js

```diff
@@ -1,31 +1,58 @@
 import React from 'react'
 //import './style.css'
 import { NavLink } from 'react-router-dom'
-import { NavWrapper } from './style.js'
+import { NavWrapper, ThemeSelector } from './style.js'
+import Select from 'react-select'
+import { themes, defaultTheme } from '../../globalStyle.js'
 
-const NavBar = ({ changeTheme }) => (
-    <NavWrapper>
-        <div>
-            <ul>
-                <li>
-                    <NavLink exact to="/" activeClassName="active">
-                        Home
-                    </NavLink>
-                </li>
-                <li>
-                    <NavLink to="/devices/" activeClassName="active">
-                        Devices
-                    </NavLink>
-                </li>
-                <li>
-                    <NavLink to="/users/" activeClassName="active">
-                        Users
-                    </NavLink>
-                </li>
-            </ul>
-            <input type="button" onClick={changeTheme} value="ChangeTheme" />
-        </div>
-    </NavWrapper>
-)
+class NavBar extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            selectedOption: null,
+        }
+    }
+
+    handleChange = selectedOption => {
+        this.setState({ selectedOption })
+        this.props.changeTheme(selectedOption.value)
+    }
+
+    render() {
+        return (
+            <NavWrapper>
+                <div>
+                    <ul>
+                        <li>
+                            <NavLink exact to="/" activeClassName="active">
+                                Home
+                            </NavLink>
+                        </li>
+                        <li>
+                            <NavLink to="/devices/" activeClassName="active">
+                                Devices
+                            </NavLink>
+                        </li>
+                        <li>
+                            <NavLink to="/users/" activeClassName="active">
+                                Users
+                            </NavLink>
+                        </li>
+                    </ul>
+                    <ThemeSelector>
```

Comment:
ThemeSelector element is redundant. It can be removed and css can be placed on Select element directly